CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in the last 3 major versions of the postg
 
 ## Unreleased
 
+## 8.1.2 - 2021-01-27
```

Comment:
Please use this header as our automated release system will fix this when it's released.
```suggestion
## Unreleased
```